openslides_backend/action/actions/motion_workflow/create.py

```diff
@@ -30,3 +33,82 @@ def get_dependent_action_payload(
                 "first_state_of_workflow_id": instance["id"],
             }
         ]
+
+
+@register_action("motion_workflow.create_simple_workflow", internal=True)
+class MotionWorkflowCreateSimpleWorkflowAction(CreateAction):
+    """
+    Action to create a motion workflow.
+    """
+
+    model = MotionWorkflow()
+    schema = DefaultSchema(MotionWorkflow()).get_create_schema(
+        ["name", "meeting_id"],
+        [
+            "default_workflow_meeting_id",
+            "default_amendment_workflow_meeting_id",
+            "default_statute_amendment_workflow_meeting_id",
+            "first_state_id",
+        ],
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Updates one instance of the payload. This can be overridden by custom
+        action classes. Meant to be called inside base_update_instance.
+        """
+        additional_relation_models: Dict[FullQualifiedId, Any] = {
+            FullQualifiedId(self.model.collection, instance["id"]): instance
+        }
+        payload = [
+            {
+                "name": "submitted",
+                "allow_create_poll": True,
+                "allow_support": True,
+                "workflow_id": instance["id"],
+                "first_state_of_workflow_id": instance["id"],
+            },
+            {
+                "name": "accepted",
+                "recommendation_label": "Acceptance",
+                "css_class": "green",
+                "merge_amendment_into_final": "do_merge",
+                "workflow_id": instance["id"],
+            },
+            {
+                "name": "rejected",
+                "recommendation_label": "Rejection",
+                "css_class": "red",
+                "merge_amendment_into_final": "do_not_merge",
+                "workflow_id": instance["id"],
+            },
+            {
+                "name": "not_decided",
+                "recommendation_label": "No decision",
+                "css_class": "grey",
+                "merge_amendment_into_final": "do_not_merge",
+                "workflow_id": instance["id"],
+            },
+        ]
+
+        write_requests, action_results = self.execute_other_action(
+            MotionStateActionSet.get_action("create"),
+            payload,
+            additional_relation_models,
+        )
+        additional_relation_models.update(
+            {
+                event["fqid"]: event["fields"]
+                for event in write_requests.events  # type: ignore
+                if event["type"] == EventType.Create
+            }
+        )
```

Comment:
Same here as above: use the `action_results` instead of the write requests

Reply:
see above